src/app/containers/Footer/__snapshots__/index.test.jsx.snap

```diff
@@ -17,30 +17,37 @@ exports[`FooterContainer should render correctly 1`] = `
       Array [
         Object {
           "href": "https://www.bbc.com/news/help-41670342",
+          "id": "trust",
```

Comment:
Curious why you're using an id here, rather than the text, it seems a tad superfluous. And could also cause issues in future.

If you navigate from 1 service to another, when we localise this data, the text could change, but the key wont, confusing react.

Reply:
@dr3 I suppose it is a tad superfluous. I'm sure we can be confident that we will never have 2 footer links with identical text. I'll remove the IDs here and use just the text as the key in the Psammead component.